### PR TITLE
updating the location of File Upload Addon

### PR DESCRIPTION
**Pull request by rdarkflow**
Signed-off-by: Rami Ahmad <rahmad@protonmail.com>

**- Summary**

**- Test plan**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: site/content/blog/2021-08-20-zap-fileupload-addon/index.md [hunk 1]
```diff
@@ -47,10 +47,10 @@ Vulnerabilities covered are:
 - Remote Code Execution by uploading JSP and its variants
 - No virus scan. 
 
-The FileUpload add-on is currently only available for dev versions of ZAP such as weekly releases or built from source, as it depends on an up-to-date version of ZAP's core.
+The FileUpload add-on is available in ZAP Marketplace.
```
Comment (psiinon) [inline site/content/blog/2021-08-20-zap-fileupload-addon/index.md:50]: "in the" or "via the" - cpuld link "ZAP Marketplace" to https://www.zaproxy.org/addons/

Comment (thc202) [reply site/content/blog/2021-08-20-zap-fileupload-addon/index.md:50]: I'd suggest keeping the previous sentence and add one that mentions the add-on will be available in the marketplace for newer ZAP versions.

**Review by psiinon — APPROVED** [review]

File: site/content/blog/2021-08-20-zap-fileupload-addon/index.md (hunk 1 above)
Comment (kingthorin) [reply site/content/blog/2021-08-20-zap-fileupload-addon/index.md]: I believe thc202's comment is still outstanding.

Comment (thc202) [reply site/content/blog/2021-08-20-zap-fileupload-addon/index.md]: To be clear I'm suggesting that to not rewrite history, the blog posts contents are accurate on/for the date they were written and one should read them taking into account that.

**Comment by thc202** [comment]
Addressed the pending comment.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**